fmcg_store/data/product_template.xml

```diff
@@ -0,0 +1,247 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="product_template_10" model="product.template" context="{'create_product_product': False}">
+        <field name="name">Rozana super Basmati Rice</field>
+        <field name="image_1920" type="base64" file="fmcg_store/static/src/binary/product_template/10-image_1920"/>
+        <field name="available_in_pos" eval="True"/>
+        <field name="detailed_type">product</field>
+        <field name="uom_id" ref="uom.product_uom_gram"/>
+        <field name="uom_po_id" ref="uom.product_uom_gram"/>
```

Comment:
Default value?
To check if we can remove it!

Reply:
it is needed